docs/api/exceptions.md

```
@@ -0,0 +1,54 @@
+```python
+from oinker import OinkerError
+
+try:
+    await piglet.dns.list("example.com")
+except OinkerError as e:
+    print(f"Something went wrong: {e}")
+```
+
+## Exception Hierarchy
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)